src/platform/surveys/useFeatureUsageTracker.test.ts

```
@@ -0,0 +1,123 @@
+    const featureB = useFeatureUsageTracker('feature-b')
+
+    featureA.trackUsage()
+    featureA.trackUsage()
+    featureB.trackUsage()
+
+    expect(featureA.useCount.value).toBe(2)
+    expect(featureB.useCount.value).toBe(1)
+  })
+
+  it('persists to localStorage', async () => {
+    const { useFeatureUsageTracker } = await import('./useFeatureUsageTracker')
```

ESLint

[error] Unable to resolve path to module './useFeatureUsageTracker'.

(import-x/no-unresolved)